cypress/support/commonCommands.js

```diff
@@ -0,0 +1,14 @@
+import "@testing-library/cypress/add-commands";
```

Comment:
This file was removed in last commit. We are not going to use commands now, as we have functions in place.